universal-login-commons/lib/core/models/message.ts

```diff
@@ -19,6 +19,15 @@ export type PaymentOptions = {
   gasToken: string;
 };
 
+export type ExecutionOptions = {
+  gasPrice: utils.BigNumberish;
+  gasLimit?: utils.BigNumberish;
+  gasToken?: string;
+  nonce?: string | number;
+};
+
+export type SdkExecutionOptions = ExecutionOptions & Required<Pick<ExecutionOptions, 'gasLimit'>>;
```

Comment:
Can be PartialRequired<ExecutionOptions. 'gasLimit' | 'gasPrice'> (I checked but up to you what is more readable)

Reply:
PartialRequired<ExecutionOptions. 'gasLimit' | 'gasPrice'> doesn't work as expected